api/e2e/gql_trigger_test.go

```
@@ -0,0 +1,184 @@
+
+	workflowContent := `{
+		"name": "Test Workflow",
+		"version": "1.0",
+		"steps": []
+	}`
+	part, err := w.CreateFormFile("0", "workflow.json")
+	assert.NoError(t, err)
+	_, err = part.Write([]byte(workflowContent))
+	assert.NoError(t, err)
+
+	w.Close()
```

golangci-lint (1.62.2)

Error return value of `w.Close` is not checked

(errcheck)